libs/langchain/langchain/graphs/neo4j_graph.py

```diff
@@ -69,11 +70,6 @@ def __init__(
                 "'apoc.meta.data()' is allowed in Neo4j configuration "
             )
 
-    @property
-    def get_schema(self) -> str:
```

Comment:
not sure if you want to leave it in and deprecate, to not break people's code?

Reply:
This was suggested in another PR since `get_schema` is a bad name for `schema` property. They can use `schema` property now.